src/README.md

```diff
@@ -49,6 +49,31 @@ var myApp = new springroll.Application({
 Note that if any of `vo`, `music`, or `sfx` are available features, `sound` will be marked as a feature implicitly.
 Also, all of these features are marked `false` by default.
 
+## Difficulty Features
+### Hit Area Scale
+Enable this feature if your game has scalable hit boxes for entities. This value will allow you to increase the size of hit boxes for potentially hard-to-hit buttons or UI elements. Caution: Prevent overlapping hit boxes for larger settings
+
+### Drag Threshold Scale
+Enable this feature to make dragging detection more sensitive and allows the distance delta to be configurable. Typically developers need to use a distance delta for the detection of a drag but it can sometimes be too large or small for users.
+
+### Health
+Enable this feature if your game does not have a configurable number of attempts, retries, lives, or health. Players may want to adjust these numbers manually to give themselves more tries at an objective or lower health to increase difficulty.
+
+### Object Count
+Enable this feature in the game to allow players to adjust the number of objects used to complete objectives. This will allow players to increase or decrease the number of items used in the game, hidden and visible.
+
+### Speed Scale
+Enable this feature if the speed of the game mechanics can increase or decrease. By slowing down or speeding up the game features the player can increase or decrease the difficulty as needed.
+
+### Completion Percentage
+Enable this feature if your game includes mechanics with a configurable completion percentage for a task. This allows the player to change the amount of times they need to interact before moving forward in the game. Example: How many swipes it takes to clear dirt off a puzzle.
+
+### Timer Scale
+Enable this feature in your game includes an adjustable timer mechanic. This allows players to give themselves more or less time to complete a challenge depending on the difficulty level they require.
+
+### Input Count
+Enable this feature if your game includes a configurable mechanic that requires multiple clicks, taps or keyboard input to complete an objective. This allows players interact with the elements more or less to complete the objective depending on their difficulty requirements
```

Comment:
`This allows players interact with...` should be `This allows players to interact with...`

Reply:
Fixed